docs/detections/rules-ui-create.asciidoc

```diff
@@ -266,9 +266,16 @@ NOTE: To preview rules, you need the `read` privilege to the `.preview.alerts-se
 To preview a rule:
 
 . Write the rule query.
+. Choose how you want to preview the query results:
+
+** *Quick query preview*: Allows you to select from three pre-defined timeframes -- *Last hour*, *Last day*, or *Last month* -- when previewing rule results. By default, rules are scheduled to run every 5 minutes with an additional look-back time of 1 minute. These settings cannot be modified.
```

Comment:
Looks great, though the default timeframes and values are a little bit more complicated.

The pre-defined timeframes are different for different rule types:

- Custom Query: hour, day, month
- Machine Learning: hour, day, month
- Threshold: hour
- Event Correlation: hour, day
- Indicator Match: hour, day, week
- New Terms: hour, day, month

Also, default "scheduled to run" and "additional look-back time" values are different for different timeframes.

- "Last hour": "scheduled to run" set to 5 minutes and "additional look-back time" set to 1 minute
- "Last day": "scheduled to run" set to 1 hour and "additional look-back time" set to 5 minutes
- "Last week": "scheduled to run" set to 1 hour and "additional look-back time" set to 5 minutes
- "Last month": "scheduled to run" set to 1 day and "additional look-back time" set to 1 hour

Reply:
Ah, I remember that the pre-defined timeframes are different for certain rule types but didn't know about the rule interval and look-back time values. Thanks for sharing this information! I'll work on integrating these next week when I'm back from PTO.